### ISSUE TITLE
Could not get CLTVersion

### ISSUE THREAD

**Issue by tibbers622** [nodejs/node-gyp#1849]
if CLT version is over 10.*, it could not get the version.
`version = re.match(r'(\d+\.\d+\.?\d*)', version).groups()`

and now latest CLT version is 10.14.

https://developer.apple.com/download/more/?=command%20line%20tools

https://github.com/nodejs/node-gyp/blob/872ee89a8541264ee2d96e162d410f478bba7815/gyp/pylib/gyp/xcode_emulation.py

**Comment by rvagg** [comment]
I don't think that's your problem @tibbers622:

```
$ python
Python 2.7.16 (default, Mar 22 2019, 15:22:50)
[GCC 4.2.1 Compatible Apple LLVM 10.0.0 (clang-1000.11.45.5)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import re
>>> version = "10.14"
>>> re.match(r'(\d+\.\d+\.?\d*)', version).groups()[0]
'10.14'
```

Can you give us some error output for a problem you're experiencing with this code? The `\d+` should match both `10` and `14` just fine.

**Comment by cclauss** [comment]
Fascinating because GYP3's code incorrectly returns 10.1

https://github.com/refack/GYP3/blob/master/gyp/xcode_emulation.py#L1407

**Comment by cclauss** [comment]
I believe that #1890 fixed this. Please reverify.

**Comment by cclauss** [comment]
@tibbers622 Should this issue be closed?

**Comment by cclauss** [comment]
Closing in favor of #1927.  @tibbers622 If this has not been resolved, please jump in on #1927.
